Makefile

```diff
@@ -3,10 +3,16 @@ check_prerequisites:
 	@echo "Python version: $(shell python --version)"
 	@echo "Poetry version: $(shell poetry --version)"
 	@echo "Npm version: $(shell npm --version)"
+	@sh ./scripts/check_hooks.sh
```

Comment:
Developer has to run `make install` and hooks would be set anyway, do we really need this check in the prerequisites?

Reply:
`check_prerequisites` writes the versions of the packages used, I think it's a good idea to inform the contributor at this stage that something not configured. Anyway, it must be done. Here, the easy way to do it